### ISSUE TITLE
Remove obligatory google objects initiation at hat-game import

### ISSUE THREAD

**Issue by aguschin** [production-ml/the-hat-game#7]
If you'll try to run `GameRun_Demo.ipynb`, you'll get:

```
DefaultCredentialsError: Could not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started
```

This happens because by default we try to initialise things which require using google account and GCS bucket. To use Hat Game, one should be setting up them. This should be an optional dependency. For example, I may want to just run the Jupyter notebooks with few games, in that case I don't need anything from google.

The error happens here: https://gitlab.com/production-ml/the-hat-game/-/blob/master/settings.py

We should improve that.

**Comment by aguschin** [comment]
As discussed with @olegpolivin, we probably should split the package:
1. Put pure Hat game logic in one package, 
2. Put server/host logic in another package.

**Comment by aguschin** [comment]
Resolved in #23
